holoviews/plotting/renderer.py

```diff
@@ -13,11 +13,17 @@
 from contextlib import contextmanager
 
 import param
+
+from panel import config
+from panel.io.notebook import load_notebook, render_model, render_mimebundle
+from panel.pane import HoloViews
```

Comment:
I would prefer to have `HoloViews` be qualified i.e used as `panel.pane.HoloViews` or be imported as `HoloViewsPane`....

Reply:
Done